apache/Readme.md

```diff
@@ -0,0 +1,22 @@
+# Apache reverse proxy server configuration
+
+#### File path
+ Be sure that the following two files are placed in the right Apache configuration folder.
+ 
+ `/etc/httpd/conf/httpd.conf`
+ 
+ `/etc/httpd/conf/httpd-le-ssl.conf`
+ 
+#### Note: These require that a valid SSL (https) certificate is loaded in the file.
+
+* Load the /new_server versions for a fresh VM install.
+    * These do not include the certificates in the ssl Apache conf.
+* Load the /live_server_fix to easily fix any corrupted files.
+ 
+```
+## SSL directives
+SSLCertificateFile /etc/letsencrypt/live/software-test.cioos-atlantic.ca/cert.pem
```

Comment:
Ensure master is cioosatlantic.ca and dev is dev.cioosatlantic.ca for any instance like this.
```suggestion
SSLCertificateFile /etc/letsencrypt/live/cioosatlantic.ca/cert.pem
```